pkg/config/config.go

```diff
@@ -79,28 +79,31 @@ type LocalConfigInfo struct {
 	LocalConfig `yaml:",omitempty"`
 }
 
-func getLocalConfigFile() (string, error) {
+func getLocalConfigFile(cfgDir string) (string, error) {
 	if env, ok := os.LookupEnv(localConfigEnvName); ok {
 		return env, nil
 	}
 
-	wd, err := os.Getwd()
-	if err != nil {
-		return "", err
+	if cfgDir == "" {
+		var err error
```

Comment:
why not use `err:=` in the next line?

Reply:
only to be explicit that cfgDir remains the same in this block as the one passed as parameter to this func

Reply:
OK :+1: